apps/frontend/src/libs/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,78 @@
+import { Icon } from "~/libs/components/components.js";
+import { useLocation } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+type BreadcrumbItem = {
+	link?: string;
+	name: string;
+};
+
+type Properties = {
+	items?: BreadcrumbItem[];
+};
+
+const FIRST_ITEM_INDEX = 0;
+const SINGLE_ITEM_OFFSET = 1;
+
+const formatBreadcrumbName = (name: string): string => {
+	return (
+		name
+			.replaceAll("-", " ")
+			// eslint-disable-next-line unicorn/prefer-string-replace-all
```

Comment:
You're right, i am working on that now, i would love any suggestions you have

Reply:
I'm not certain, but it might be something like this:
```
const formatBreadcrumbName = (name: string): string => {
	return name
		.split("-")
		.map((word) => word.charAt(0).toUpperCase() + word.slice(1))
		.join(" ");
};
```
I did not test it, it looks a bit more bulky than yours, but it'll do. (P.S. if using this, you'll need to deal with magic numbers).  Also, I agree with Mykhailo, it surely does look like a helper.

Reply:
I think you can keep:
```ts
const formatBreadcrumbName = (name: string): string =>
		name
			.replaceAll("-", " ")
			.replaceAll(/\b\w/g, (char) => char.toUpperCase());
```
as I mentioned [here](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/104/files/153faf35e7bbcc622e619b299caae24e44a5b52c#r1734259823)

Reply:
Wow. Github goes crazy with tabulation size... As I remember, previously tab-size was displayed with about two spaces, but now it looks like there 8 spaces per tab...